rosdep/python.yaml

```diff
@@ -5969,6 +5969,10 @@ python3-matplotlib:
     '7': null
   slackware: [python3-matplotlib]
   ubuntu: [python3-matplotlib]
+python3-mechanize:
+  debian: [python3-mechanize]
```

Comment:
It looks like `python3-mechanize` is only available in bullseye and later, but we still support `buster`.  So I'll suggest:

```
debian:
  '*': [python3-mechanize]
  buster: null
```